public/app/diarization-html-snippet.html

```
@@ -0,0 +1,97 @@
+     DIARIZATION TIMELINE + ISOLATION CONTROLS
+     Paste this block inside <main> in public/app/index.html,
+     directly after the existing waveform section.
+──────────────────────────────────────────────────────────────── -->
+
+<!-- ── Diarization Timeline ───────────────────────────────────── -->
+<section id="diarization-section" class="vip-panel diarization-section" data-requires-tier="PRO">
+  <div class="vip-panel__header">
+    <span class="vip-panel__title">🎙 Speaker Timeline</span>
+    <div class="diarization-toolbar">
+      <span id="diarization-time-label" class="diarization-time-label">00:00.0</span>
+      <button id="diarization-zoom-in"  class="vip-btn vip-btn--sm" title="Zoom in">＋</button>
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)